html.mjs

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * Creates a link with report results.
+ * @param {string} url URL to the result.
+ * @return {string} A link to the results.
+*/
+export const getResultsLink = (url) => (`
```

Comment:
let's use regular functions here. I don't think the arrow functions and lack of `return` gives us much.